packages/babylon/package.json

```diff
@@ -25,6 +25,8 @@
   "devDependencies": {
     "@babel/helper-fixtures": "7.0.0-beta.31",
     "babel-plugin-transform-for-of-as-array": "1.0.4",
+    "babel-plugin-transform-charcodes": "0.0.7",
+    "charcodes": "0.0.6",
```

Comment:
This should be `0.0.8`. Also, maybe https://github.com/xtuc/charcodes/blob/c08d3c5906475f0e2e7b7d2b316c8abdef526e02/packages/babel-plugin-transform-charcodes/package.json#L45 should be a per dependency?

Reply:
Yes right. The transform doesn't require a specific version of `charCodes`, it's probably a good idea to use a peer dependency here.

Reply:
Ok I updated with ^